strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -187,8 +187,10 @@ def init_circuit(self, prog, **kwargs):
         self.circuit = BosonicModes()
         init_weights, init_means, init_covs = [[0] * nmodes for i in range(3)]
 
-        vac_means = np.zeros(2, dtype=complex).tolist()
-        vac_covs = np.identity(2, dtype=complex).tolist()
+        vac_means = np.zeros((1, 2), dtype=complex)  # .tolist()
+        vac_covs = np.array(
+            [[[0.5 * self.circuit.hbar, 0], [0, 0.5 * self.circuit.hbar]]]
```

Comment:
maybe a bit clearer/shorter to write `np.array([0.5*self.circuit.hbar*np.identity(2)])` ?